### PR TITLE
feature: keyword recognizer uses json metadata for classes if available

### PR DESCRIPTION
**Pull request by noelweichbrodt**

### CODE REVIEW

File: Spokestack/TFLiteKeywordRecognizer.swift [hunk 1]
```diff
@@ -72,7 +72,16 @@ import TensorFlowLite
         self.encodeState.fill(0.0) // fill now because the encoded state is used to feed-forward
         let encodeLength = c.keywordEncodeLength * c.sampleRate / 1000 / self.hopLength
         self.encodeWindow = RingBuffer(encodeLength * c.keywordEncodeWidth, repeating: -1.0)
-        self.classes = self.configuration.keywords.components(separatedBy: ",")
+        do {
+            let keywords = try CommandModelMeta(configuration)
+            self.classes = keywords.model.classes.flatMap({ c in
```
Comment (space-pope) [inline Spokestack/TFLiteKeywordRecognizer.swift:77]: We don't want the utterance text here, just the class names—the model outputs will correspond to training classes, not individual utterances that resolve to those classes.

**Review by space-pope — CHANGES_REQUESTED** [review]
There's an easy-to-make misunderstanding about the metadata here, noted below. I'd double-check the test metadata itself, but GitHub won't show it to me since it's tracked by git-lfs (I only do that for the model files themselves, fwiw).

File: Spokestack/TFLiteKeywordRecognizer.swift (hunk 1 above)
Comment (noelweichbrodt) [reply Spokestack/TFLiteKeywordRecognizer.swift:77]: 🤦  good catch, thanks

**Review by space-pope — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**